### PR TITLE
Tweak wording for Volume concept page

### PR DESCRIPTION
**Pull request by seancrasto**
Grammatical corrections.

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ the {{< glossary_tooltip text="container image" term_id="image" >}}, plus volume
 (if defined) mounted inside the container.
 The process sees a root filesystem that initially matches the contents of the container
 image.
-Any writes to within that filesystem hierarchy, if allowed, affect what that process views
-when it performs a subsequent filesystem access.
+Any writes within that filesystem hierarchy, if allowed, affects what that process views
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/storage/volumes.md:52]: Changes look good to me.

Deploy preview: https://deploy-preview-39503--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/

**Comment by bradtopol** [comment]
/assign
I have requested some minor grammatical updates

**Comment by natalisucks** [comment]
@seancrasto Hey there 👋 Are you able to address the reviewer feedback and rebase the PR due to the checks failing? That way, we can get to reviewing and hopefully merging. Thanks!

**Comment by divya-mohan0209** [comment]
Hello @seancrasto , thank you for your efforts & this PR. However, due to lack of activity from your side on this PR since March, we'll be closing this out. If you wish to resume work on the PR, please feel free to reopen the PR with a /reopen command, address the suggestions and the merge conflict.
/close

**Pull request closed without merging**